services/src/main/java/org/keycloak/keys/AbstractEcKeyProviderFactory.java

```diff
@@ -58,7 +47,7 @@ public void validateConfiguration(KeycloakSession session, RealmModel realm, Com
                 .checkBoolean(Attributes.ACTIVE_PROPERTY, false);
     }
 
-    public static KeyPair generateEcdsaKeyPair(String keySpecName) {
+    public static KeyPair generateEcKeyPair(String keySpecName) {
```

Comment:
I would also change here to remove the `SHA1PRNG` and create the EC key without passing the ransom. But this was done before your change so this can continue this way.